### PR TITLE
Testing new ECS setup in lab

### PR DESCRIPTION
**Pull request by Omicron7**

### CODE REVIEW

**Comment by Omicron7** [comment]
I've fixed conflicts and merged this to staging. I'll squash then when merging to master, then reset staging.

File: .github/workflows/build-deploy-ecs.yml [hunk 1]
```diff
@@ -5,8 +5,9 @@ on:
     branches:
       # Automatically build and deploy master and staging. Additional branches may be added.
       # rails-infrastructure-canary doesn't have production infrastructure, so don't build it there.
-      # - master
+      - master
       - staging
+      - lab-ecs-v2
```
Comment (jbirdjavi) [inline .github/workflows/build-deploy-ecs.yml:10]: Are we still using lab? Is production actually building now?

File: config/environments/development.rb [hunk 2]
```diff
@@ -3,6 +3,14 @@
...
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Whitelist lab hostname
+  config.hosts << "rails-canary-lab.aws.cru.org"
+
+  # Web Console
+  # https://github.com/rails/web-console
+  config.web_console.whiny_requests = false
+  config.web_console.permissions = "10.0.0.0/8"
+
```
Comment (jbirdjavi) [inline config/environments/development.rb:13]: Are these purely related to the lab instance?

**Review by jbirdjavi — APPROVED** [review]
Changes needed if lab is going away and/or if production is still not building, otherwise good I think.

File: config/environments/development.rb (hunk 2 above)
Comment (Omicron7) [reply config/environments/development.rb:13]: The config.hosts was due to the dns remapping that's enabled in development mode, which is what `lab` ran as.

I believe I set the others based on console/log output warning about them not being set in the lab/dev testing. I don't think it hurts anything to leave them.

**Review by jbirdjavi — APPROVED** [review]
